extension/src/background/helpers/dataStorage.ts

```diff
@@ -0,0 +1,36 @@
+import { browser } from "webextension-polyfill-ts";
+
+interface SetItemParams {
+  [key: string]: any;
+}
+
+export const dataStorage = {
+  getItem: async (key: null | string | string[] | { [s: string]: any }) => {
+    // Passing an object (for ex: {myKey: "aValue"}) will return the key's (myKey) value from storage.
+    // If the key is not found in storage, it will return the the value passed along in object as a default (in this case, "aValue")
+
+    const storageResult = await browser.storage.local.get(key);
+
+    return storageResult;
+  },
+  setItem: async (setItemParams: SetItemParams) => {
+    await browser.storage.local.set(setItemParams);
+  },
+};
```

Comment:
this is the new storage API wrapper. We'll eventually replace all instances of `localStorage` with `dataStorage`